tests/utils/index.js

```diff
@@ -1562,6 +1553,9 @@ const isMinimumChromeVersion = process.env.CHROME_VERSION === MINIMUM_BROWSER_VE
 
 const escapeBranchName = (branch) => branch?.replace(/[/|_]/g, '-');
 
+const toggleSentinelTransitions = () => sendSignal('sentinel', 'USR1');
```

Comment:
I'm not sure about toggle. Wouldn't it be better to have an explicit start and an explicit stop? It would be very easy to accidentally call toggle twice (eg: test retry) which leaves sentinel in an unexpected state. This would be very hard to debug because the double call could be in one test, but the failure only show up many tests later. I think having two separate signals would make it explicit. Then we can decide if calling `startProcessing` when already processing would be ignored, logged, or throw an error because someone forgot to clean up.

Reply:
As above, we can't send custom signals.